core/src/test/scala/org/apache/spark/sql/test/generator/Schema.scala

```diff
@@ -49,6 +45,24 @@ case class Schema(database: String,
 
   assert(indexInfo.count(_.isPrimary) <= 1, "more than one primary key exist in schema")
 
+  private val pkIndexInfo = indexInfo.filter(_.isPrimary)
```

Comment:
primary key defined in index implies primary key is not handle. Have we considered primary key is handle case here?

Reply:
Never mind. I mislead by the name of `indexInfo`.